test/test_foreach.py

```diff
@@ -619,9 +605,9 @@ def test_add_list_error_cases(self, device):
 
         # One empty list
         tensors1.append(torch.tensor([1], device=device))
-        with self.assertRaisesRegex(RuntimeError, "Tensor list must have at least one tensor."):
+        with self.assertRaisesRegex(RuntimeError, "Scalars list must have at least one value."):
```

Comment:
Empty list gets resolved to Scalar list? That might be confusing, but ok. What is the error message for 2 empty lists?

Reply:
Updated test case for 2 empty lists to have text.